openwisp-utils-qa-checks

```diff
@@ -93,17 +96,32 @@ runcheckcommit() {
   fi
 }
 
+runmakemigrationcheck() {
+  echo "Running Make Migrations Check"
+  if [ ! -f "./tests/manage.py" ]; then
+    echo "File manage.py not found\nSkipping Make Migration Check"
+  else
+    OUTPUT=$(python tests/manage.py makemigrations --dry-run $MODULE)
+    echo $OUTPUT
+    echo $OUTPUT | grep "No changes detected" \
+      && echo "SUCCESS: Migrations check successful!" \
+      || { echoerr "ERROR: Make Migrations check failed!"; FAILURE=1; }
```

Comment:
Some GCI students have a hard time understanding migrations, so can you please make the error statement more discriptive, like: 
```
"ERROR: Migrations check failed! Models' changes not migrated, please run `./manage.py makemigrations` to solve the issue!"
```